internal/cmd/tokens.go

```diff
@@ -51,16 +47,6 @@ func tokensCreate() error {
 
 	token, err := client.CreateToken()
 	if err != nil {
-		if errors.Is(err, api.ErrForbidden) {
```

Comment:
the tokens subcommand already checks if the session is valid so this is not useful